src/back-end/lib/db.ts

```diff
@@ -351,35 +352,42 @@ export async function readOneAffiliation(connection: Connection, user: Id, organ
 
 export async function readOneAffiliationById(connection: Connection, id: Id): Promise<Affiliation> {
   const result = await connection('affiliations')
-    .where({
-      id
-    })
+    .where({ id })
+    .andWhereNot({ membershipStatus: MembershipStatus.Inactive })
```

Comment:
@sutherlanda FYI this `andWhereNot` function call was missing in several affiliation functions in `db.ts`

Reply:
It resulted in a bug that would vend inactive affiliations to API consumers.

Reply:
Got it, thanks for correcting.